src/figuresClasses.ts

```diff
@@ -1,11 +1,74 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: Figure['shape'] = 'triangle';
+
+  constructor(
+    public color: Figure['color'],
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
Since you're using TypeScript, you can take advantage of parameter properties to simplify your code. TypeScript allows declaring and initializing class members directly in the constructor parameters by using access modifiers like 'public' or 'private'. In this case, you don't need to manually assign the parameters to class properties as it's done implicitly.